clients/admin-ui/src/features/system/system-form-declaration-tab/PrivacyDeclarationForm.tsx

```diff
@@ -250,6 +261,7 @@ export const PrivacyDeclarationFormComponents = ({
           variant="stacked"
           options={[]}
           disableMenu
+          disabled={lockedForGVL}
```

Comment:
hm this one doesn't seem to be working, I can still add stuff 
![image](https://github.com/ethyca/fides/assets/24641006/d66bb45c-2c36-41fc-b68a-e2e60d1fdc92)

Reply:
I wonder if the save button should go away—I was even able to save "this is new" above

Reply:
I can also delete data uses, which I don't think we want?

Reply:
Got it, I have removed the delete button and the save button inside the modal when the form is locked.  The issue with Features not being appropriately disabled was just a naming error, it was looking for `isDisabled` instead of `disabled`.